client/src/components/Accordion/Accordions.js

```diff
@@ -91,6 +90,8 @@ class AccordionEnterExit extends React.Component {
 
 AccordionEnterExit.defaultProps = {
   max_height: "80vh",
+  opening_opacity: 1e-6,
+  closing_opacity: 1,
```

Comment:
I'm not sure if we'd want to have the animation opacities be custom (or what use-cases would that be)

I'll probably let someone else provide some input on this haha